packages/website/components/accountPlansModal/accountPlansModal.js

```diff
@@ -76,10 +80,28 @@ const AccountPlansModal = ({
           </div>
         )}
 
+        <div className="billing-card card-transparent">
+          <div className="billing-terms-toggle">
+            <input
+              type="checkbox"
+              id="agreeTerms"
+              checked={hasAcceptedTerms}
+              onChange={() => setHasAcceptedTerms(!hasAcceptedTerms)}
+            />
+            <label htmlFor="agreeTerms">
+              I agree to the{' '}
+              <a href="https://web3.storage/terms/" rel="noreferrer" target="_blank">
```

Comment:
We should probably use a url relative to this page so people see the version of that page from the same source code powering this page.

```suggestion
              <a href="/terms/" rel="noreferrer" target="_blank">
```